domains/seldomgreen.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
+           "discord": "761646485334523954"
+        },
+    
+        "record": {
+            "CNAME": "is-a.dev"
```

Comment:
Could you provide the reason as to why you want this domain to point to our homepage?